core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>Javascript Not Enabled</h2>
+                  <p>Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires Javascript to be enabled in your web browser in order to function properly and your web browser has Javascript disabled.  To learn more about Oppia, you need to <a href="http://www.enable-javascript.com/">enable Javascript.</a></p>
```

Comment:
For the last sentence, it might be better if we said something like:
`If you need help enabling Javascript for your web browser, click here`. 
The `here` will link the user to enable-javascript.com. (This is so that the user doesn't assume that the link will enable JS for them)